core/src/main/scala/org/apache/spark/h2o/converters/SupportedRDDConverter.scala

```diff
@@ -29,7 +29,7 @@ import scala.reflect.runtime.universe._
   * This converter just wraps the existing RDD converters and hides the internal RDD converters
   */
 
-trait SupportedRDDConverter{
+trait SupportedRDDConverter extends Serializable{
```

Comment:
Scala creates bunch of java classes and one of those needs to be Serialisable because otherwise Spark complains that task is not serializable